src/Analyzers/Analyzers/src/WellKnownFeatures.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Analyzers
+{
+    internal static class WellKnownFeatures
```

Comment:
The plan is to expand the set of features we know about. For now just SignalR on the menu.